source/jobs/data-verification/use-data-verification.txt

```diff
@@ -1,3 +1,115 @@
+.. _rm-use-data-verification:
+
 =====================
 Use Data Verification
-=====================
\ No newline at end of file
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Data verification runs as a separate process which is tightly
+coupled to a sync job. Data verification can be enabled for one-time 
+sync jobs. This page details how to enable and debug the 
+data verification process.
+
+Enable Data Verification
+------------------------
+
+You can enable data verification when :ref:`creating a sync job <rm-create-jobs>`
+on the :guilabel:`Data Migration` tab.
+
+On the :guilabel:`Migration Options` module, click the 
+:guilabel:`Verify migrated data` check box for your sync job.
+
+
```

Comment:
(image)
is it possible to decrease the size of this image a bit.

Reply:
I looked, I think the only thing you can control is the figwidth. I also think we can remove the image entirely.